docker/init-local.sh

```
@@ -0,0 +1,55 @@
+sed -i '/redis/d' ./Procfile
+sed -i '/watch/d' ./Procfile
+
+# Get ERPNext from GitHub (required dependency)
+bench get-app erpnext
+
+# Link local HRMS app instead of getting from GitHub
+echo "Linking local HRMS app..."
+ln -sf /workspace/hrms ./apps/hrms
+
+# Create app hooks and setup
+cd ./apps/hrms
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)